shell.py

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+
+
+sys.path.insert(0, os.path.dirname(__file__))
+
+
+from yolapy.services import Yola
+from tests.test_integration.test_settings import url, auth
```

Comment:
This seems like the type of helper that shouldn't live in the root of the source.

I do like the utility.  Thoughts on moving it to `__main__.py` in yolapy and calling it via?:
```
python -m yolapy
```